### ISSUE TITLE
Question about linear finetune

### ISSUE THREAD

**Issue by tangyuelm** [leftthomas/SimCLR#22]
Thanks so much for the codes. However, I am a little bit confused about the linear finetune. In the linear.py, you used the original Cifar10 dataset to finetune the classifier. So it becomes supervised learning. To my understanding, all the training processes( including pre-train and finetune) should be based on the fact that there is little labeled data. 
 train_data = CIFAR10(root='data', train=True, transform=utils.train_transform, download=True)
In B.5 of the original paper, it says using 10% labeled data. 
I am still not sure how to fine-tune before evaluation. Should I only use a subset of CIFAR10 with labels, or use the full CIFAR10 but only 10% of data are labeled?

**Comment by leftthomas** [comment]
@tangyuelm `using 10% labeled data` it should be called weakly supervised learning, according the paper, it not only provided supervised learning results, but also weakly supervised learning results, there are a lot of tables.
